zombienet/old/0000-validator-disabling/0000-validator-disabling.toml

```diff
@@ -21,21 +21,21 @@ requests = { memory = "2G", cpu = "1" }
 #  [[relaychain.node_groups]]
 #  name = "honest-validator"
 #  count = 3
-#  args = ["-lparachain=debug"]
+#  args = ["-lparachain=trace", "--parachain-single-process"]
```

Comment:
Probably the redundant configuration. Should it be removed?
Also the same for lines 33-38